apps/course-authoring/src/renderer/pages/editor/routes.tsx

```diff
@@ -0,0 +1,21 @@
+import pageStructure from './structure';
+import pageBuild from './build';
+
+export const editorRoutes = {
+  pages: [pageStructure, pageBuild],
+};
+
+export const navigationLinks = [
+  {
+    label: 'Home',
+    link: '/',
+  },
+  {
+    label: 'Structure',
+    link: '/editor/structure',
```

Comment:
Can you change the link here and for Build so that it inherits from the imported pages

Reply:
Done. I've changed it to use the same `editorRoutes` object.